Animations/Animations.js

```diff
@@ -0,0 +1,183 @@
+                easing = 'cubic-bezier(0.165, 0.840, 0.440, 1.000)';
+              }
+              else if (easing === 'easeOutQuart'){
+                easing = 'cubic-bezier(0.230, 1.000, 0.320, 1.000)';
+              }
+              else if (easing === 'easeOutSine'){
+                easing = 'cubic-bezier(0.390, 0.575, 0.565, 1.000)';
+              }
+              else if (easing === 'easeOutExpo'){
+                easing = 'cubic-bezier(0.190, 1.000, 0.220, 1.000)';
+              }
+              else if (easing === 'easeOutCirc'){
+                easing = 'cubic-bezier(0.075, 0.820, 0.165, 1.000)';
+              }
+              else if (easing === 'easeOutBack'){
+                easing = 'cubic-bezier(0.175, 0.885, 0.320, 1.275)';
+              }
+              else if (easing === 'easeInOutQuad'){
+                easing = 'cubic-bezier(0.455, 0.030, 0.515, 0.955)';
+              }
+              else if (easing === 'easeInOutCubic'){
+                easing = 'cubic-bezier(0.645, 0.045, 0.355, 1.000)';
+              }
+              else if (easing === 'easeInOutQuart'){
+                easing = 'cubic-bezier(0.770, 0.000, 0.175, 1.000)';
+              }
+              else if (easing === 'easeInOutQuint'){
+                easing = 'cubic-bezier(0.860, 0.000, 0.070, 1.000)';
+              }
+              else if (easing === 'easeInOutSine'){
+                easing = 'cubic-bezier(0.445, 0.050, 0.550, 0.950)';
+              }
+              else if (easing === 'easeInOutExpo'){
+                easing = 'cubic-bezier(1.000, 0.000, 0.000, 1.000)';
+              }
+              else if (easing === 'easeInOutCirc'){
+                easing = 'cubic-bezier(0.785, 0.135, 0.150, 0.860)';
+              }
+              else if (easing === 'easeInOutBack'){
+                easing = 'cubic-bezier(0.680, -0.550, 0.265, 1.550)';
+              }
+
+              transitions[action]['transition-timing-function'] =  easing;
+            }
+
+            //delay--------------------------------------------
+            var delay = this.props.delay;
+            //default if no delay prop
+            if (delay){
+              //case to accept if number is passed into delay
+              if (typeof delay === "number") {
+                delay = delay + "ms"
+              }
+              transitions[action]['transition-delay'] =  delay;
+            }
+
+            //custom properties---------------------------------
+            var custom = this.props.custom;
+            if (custom){
+              for (cssElement in custom) {
+                transitions['enterActive'][cssElement] =  custom[cssElement];
+                transitions['leave'][cssElement] =  custom[cssElement];
+              }
+            }
+          }
+        }//end for loop for transitions
+
+      //return new React.Dom element
+      return (
+        React.DOM.div(
+          {className: displayName,
```

Comment:
Put the key/value pair on the next line. eg: 
{
  className: displayName,
  key: this.props.key
}